domain/src/main/java/org/oppia/android/domain/state/StateDeck.kt

```diff
@@ -172,6 +186,37 @@ internal class StateDeck internal constructor(
       .build()
   }
 
+  /**
+   * @return an [ExplorationCheckpoint] which contains all the latest values of variables of
```

Comment:
KDocs should always have a base summary fragment. Suggest just saying something like:

```kotlin
/**
 * Returns an [ExplorationCheckpoint] ...
 */
```

Also, note that spacing continuation for an at-clause is +4 spaces.

Reply:
done.
I will keep this in mind in future.